public/app/plugins/panel/piechart/plugin.json

```diff
@@ -0,0 +1,18 @@
+{
+  "type": "panel",
+  "name": "PieChart",
+  "id": "piechart",
+
```

Comment:
can we add a "state": "alpha"  here? Then we can merge this to master without it being "finished" . To be able to use alpha panels you need to modify you ini file to include:

[panels]
enable_alpha = true